airbyte-integrations/connectors/source-hubspot/acceptance-test-config.yml

```diff
@@ -41,6 +41,8 @@ acceptance_tests:
             bypass_reason: Unable to populate
           - name: deals_archived
             bypass_reason: Unable to populate
+          - name: owners_archived
+            bypass_reason: unable to populate
```

Comment:
why this can not be populated?

Reply:
right now it was not possible to archive any owners in our test account from UI or API endpoint, so this stream would be bypassed for now